cypress/integration/table/table.spec.ts

```diff
@@ -155,4 +170,137 @@ describe('Table Component', () => {
         });
     });
   });
+
+  it.skip('should be able to resize a column', () => {
+    let columnWidth = 0;
+
+    cy.get('[role="columnheader"]')
+      .first()
+      .should(($column) => {
+        const { width } = $column[0].getBoundingClientRect();
+        columnWidth = width;
+      });
+
+    cy.get('[role="separator"]')
+      .first()
+      .trigger('mousedown')
+      .trigger('mousemove', { clientX: 200, force: true })
+      .trigger('mouseup');
+
+    cy.get('[role="columnheader"]')
+      .first()
+      .should(($column) => {
+        const { width } = $column[0].getBoundingClientRect();
+        expect(width).to.be.greaterThan(columnWidth);
+      });
+  });
+
+  describe('should be able to sort by', () => {
```

Comment:
If we're mocking the data - is there any point doing this sort of e2e test? As it's not actually sorting anything right? We already test that we can sort in the unit tests

Reply:
I guess, I think I just added these for when we do need them, then they're ready to be changed for the appropriate asserts. I can remove or skip if you want

Reply:
Eh, if they work they can be left in - I was just thinking if we're not using the API in the e2e tests then these are pointless 😅

Reply:
I might as well skip them then, rather not waste time running through them